joblib/func_inspect.py

```diff
@@ -11,11 +11,15 @@
 import warnings
 import re
 import os
-
+import sys
 from ._compat import _basestring
 from .logger import pformat
 from ._memory_helpers import open_py_source
 
+
+PY3 = sys.version_info[0] >= 3
```

Comment:
Agreed, I'll do that in master directly once this PR is merged (sorry I didn't see this comment originally).